rfc-079-script-imports-of-production-data.md

```diff
@@ -0,0 +1,56 @@
+# Script imports of production data
+
+## Summary
+
+Developers writing production data to file on a local or remote server MUST do
+this via an automated script, rather than doing this manually, e.g. via the
+Rails console.
+
+This script MUST anonymise any sensitive data, and the data dump MUST be stored
+outside of the working repository to ensure it can't be inadvertedly committed
+alongside any changes to the codebase.
+
+## Problem
+
+Fetching data from a database by hand is error-prone. A data dump may be left
+on a server, committed to a repository or even in some cases alter production
+data in unforeseen ways. Developers may not realise they have exposed or
+impacted production data until long after an incident occurs.
+
+By writing a script to automate this process, we ensure that every import of
+data from a production environment is done in the same way, minimising room for
+error. It also allows us to scrub the resulting data dump of any
+personally-identifying and security-sensitive information.
```

Comment:
I've slightly re-worded this, and think the following paragraph covers the "peer review on the specifics", but let me know if it could still be clearer.

Reply:
Looks good.